index.bs

```diff
@@ -1588,6 +1588,7 @@ session.CapabilityRequest = {
   ? platformName: text,
   ? proxy: session.ProxyConfiguration,
   ? unhandledPromptBehavior: session.UserPromptHandler,
+  ? extensions: [*session.Extension],
```

Comment:
I'd recommend making this a separate PR, because you need different supporting infrastructure if we want to allow a new session request to install extensions.